inc/uninstall.class.php

```diff
@@ -510,7 +511,7 @@ static function deleteFusionInventoryLink($itemtype, $items_id) {
     *
     * @return nothing
    **/
-   static function deleteGlpiInventoryLink($item) {
+   static function deleteGlpiInventoryLink(&$item) {
```

Comment:
Objects are always passed by reference.
```suggestion
   static function deleteGlpiInventoryLink($item) {
```